MFE/devices/package.json

```diff
@@ -21,6 +21,8 @@
     "@material-ui/icons": "^4.11.2",
     "@material-ui/styles": "^4.11.3",
     "axios": "^0.26.1",
+    "b64-to-blob": "^1.2.19",
```

Comment:
I searched about the `b64-to-blob` and I found some things that I want to share:

- Last publish 5 years ago
- License is Creative Commons (I don't know if this is a problem for us)

![image](https://user-images.githubusercontent.com/36794274/181517041-03ef51be-d2f5-45b3-a7d9-e5faa378f0f6.png)

---

Searching "How to create a blob from base64" I found this answer on StackOverflow:
https://stackoverflow.com/questions/16245767/creating-a-blob-from-a-base64-string-in-javascript

There is a way to do the same as `b64-to-blob` using `window.fetch` or `axios`.

![image](https://user-images.githubusercontent.com/36794274/181516604-6e0d74fe-a14d-4ae7-a054-0b8689e3459b.png)

![image](https://user-images.githubusercontent.com/36794274/181518808-301d41ab-876b-438f-abb0-57950e6b7afd.png)

---

@luisfelipefurlan @LeonamMoura 
**My question is:** Should we use `fetch`, `axios`, or `b64-to-blob`?